README.md

```diff
@@ -75,9 +75,12 @@ NOTE:
 | CephFS | Dynamically provision, de-provision File mode RWO volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=14.2.2)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWX volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
-|        | Creating and deleting snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
+|        | Creating and deleting snapshot                            | Alpha          | >= v3.1.0          |  >= v1.0.0       | Nautilus (>v14.2.10) | >= v1.17
```

Comment:
do we support snapshot and cloning in v14.2.10 :thinking:

Reply:
madhu didnt I mention >14.2.10 ?

Reply:
sorry, I missed it. but still, as we have not tested with Nautilus isn't it good to mention Octopus?

Reply:
yeah, that can be done too.. :+1:

Reply:
@Madhu-1 mentioned octopus.. ptal.